Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,173 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/grid_2"
+      android:layout_marginEnd="@dimen/grid_2"
+      android:layout_marginHorizontal="@dimen/grid_0_5"
+      android:layout_marginVertical="@dimen/grid_4">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/email_app_bar_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/grid_2"
```

Comment:
These `paddingTop` and `paddingBottom` can be combined into `paddingVertical`.

Also, it looks a little large to me? Maybe it's `grid1` + some `paddingTop` on the nested scroll view?

Reply:
nvm! ignore my comment - Hunter found that AAPT (the build system) automatically replaces `paddingVertical` with `paddingTop`/`paddingBottom` for older devices.